tests/Module/Order/PacketSynchronizerTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace Tests\Module\Order;
+
+use Packetery\Core\Api\Soap\Client;
+use Packetery\Core\Api\Soap\Response\PacketStatus;
+use Packetery\Core\Entity\Order;
+use Packetery\Core\Log\ILogger;
+use Packetery\Module\Exception\InvalidPasswordException;
+use Packetery\Module\Options\OptionsProvider;
+use Packetery\Module\Order\PacketSynchronizer;
+use Packetery\Module\Order\Repository;
+use Packetery\Module\Order\WcOrderActions;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Tests\Core\DummyFactory;
+
+class PacketSynchronizerTest extends TestCase {
+
+	private PacketSynchronizer $packetSynchronizer;
+	private Client|MockObject $client;
+
+	protected function setUp(): void {
```

Comment:
Každý ten test je napsaný jinak, jak to tedy chceme?

Reply:
Dobrý point 👍 budeme se tedy asi řídit interními standarty. Upravím :)